### DISCUSSION TITLE
Update package on Conda-Forge

### DISCUSSION THREAD

**Discussion by JustinWinthers** [moj-analytical-services/splink#960] [Ideas]
Hi Robin,

Is it possible to get the latest stable version updated in conda-forge?  Our cyber security team prefers conda-forge, and it’s easier for us to get it approved this way.  

Thanks,

Justin

**Comment by RobinL** [comment]
Hi Justin.  The current version on conda was actually uploaded by someone else (i.e. not any of the Splink Devs) - I've never actually tried to get something on conda.  I'm off on paternity leave at the moment so unlikely have the bandwidth to look into doing this in the short term.  Can anyone do it?  I'd be perfectly happy for you to if you're able!

Thanks

Robin

**Comment by RobinL** [comment]
@JustinWinthers this is done today thanks to @ADBond .  Andy says:

>>> I’ve updated the conda-forge splink feedstock for splink, so that it is now available once again using conda (which hasn’t been the case since v.1.0.6). From now on new versions should be automatically picked up (and this worked okay for 3.5.4), although some minor manual adjustments may need to be done in certain cases (such as changing dependencies, for example). I’ll add something to the splink README to indicate this alternative installation method.
I’ve also added @Tom Hepworth as a maintainer to the relevant repo - if anyone else is keen to be added to the maintainer list then you can follow these instructions and Tom or I can approve.
